src/navigator.js

```diff
@@ -162,7 +162,7 @@ $.Navigator = function( options ){
     this.displayRegion.className = 'displayregion';
 
     (function( style, borderWidth ){
-        style.position      = 'relative';
+        style.position = 'relative';
```

Comment:
Please update this so it matches the original. As you can see here, you've removed some spaces next to the equals sign.